content/vendor/guides/ci-cd-integration.md

```diff
@@ -0,0 +1,134 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "CI/CD Integration"
+weight: "1007"
+---
+
+The KOTS CI/CD Integration guide is an advanced workflow that allows you to use a similar workflow to [Kots Starter GitHub Actions](https://github.com/replicatedhq/replicated-starter-kots/blob/master/.github/workflows/main.yml#L21) but for your own internal git repository and CI/CD systems like GitHub, GitLab, CircleCI, TravisCI, Jenkins, etc.
+
+It is broken into three sections:
+
+- [Environment Variables](#environment-variables)
+- [Kubernetes Objects](#kubernetes-objects)
+- [Helm Chart](#helm-chart)
+
+## Environment Variables
+
+This section will assume you've already completed the [Automating Your Workflow](/vendor/guides/quickstart/#automating-your-workflow) and set up a local git repo to manage your application.
```

Comment:
This line doesn't really seem to mesh with the rest of the article. Is the goal of this article to get me to a local git repo?

Reply:
Updated to better reflect what is actually needed, which is the API Token.